Menus/InventoryMenuGUI.cpp

```diff
@@ -385,7 +385,7 @@ namespace RTE {
 
 		switch (m_MenuMode) {
 			case MenuMode::Carousel:
-				if (!m_InventoryActor || (m_InventoryActor->IsInventoryEmpty() && m_InventoryActorEquippedItems.empty() && m_EnabledState != EnabledState::Disabling)) {
+				if (!m_InventoryActor || m_InventoryActor->IsInventoryEmpty() || (m_InventoryActorEquippedItems.empty() && m_EnabledState != EnabledState::Disabling)) {
```

Comment:
This isn't a proper CTD fix, and it introduces another bug, so disregard for now.

This is based on the possibility that when the actor dies, and the pie menu is open, it tries to render the pie menu using stale memory. It loops through the equipped items to render them too, but that causes a crash because they've potentially been deleted.

This fix as such doesn't truly fix it, is just makes it less likely (when the actor dies, it clears it's inventory, so m_InventoryActor->IsInventoryEmpty() should return true, so the game shouldn't crash... but this is not necessarily the case, as m_InventoryActor is still a stale pointer and IsInventoryEmpty() could still return false if another object has been allocated on top of it by the time this code runs.

This change also means that the object carousel isn't shown if all items are equipped, which is another bug we'd rather avoid.

As such, the proper fix should be to actually clear m_InventoryActor to nullptr when the actor dies.